src/bereikbaarheid/bollards/bollards.py

```diff
@@ -0,0 +1,237 @@
+from django.db import connection
+
+from bereikbaarheid.utils import django_query_db
+
+"""
+    The query calculates a route to the provided lat/lon and returns
+    the encountered bollards. If no bollards are found, nothing is returned.
+
+    It works as follows:
+    - Based on the provided day of the week, start and end time,
+        the network cost is recalculated:
+        - cost is multiplied by 10000 if:
+        - road section is linked to a bollard
```

Comment:
Nitpicking here, maar ik zou deze indentation aanpassen zodat duidelijker is van waar tot waar de if is.